web/modules/custom/greet/src/Controller/GreetController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\greet\Controller\GreetController.
+ */
+namespace Drupal\greet\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+class GreetController extends ControllerBase {
```

Comment:
Please add proper class docblock.

Reply:
I have added the class docblock for this.